superset/sql_parse.py

```diff
@@ -765,8 +765,8 @@ def _extract_tables_from_sql(self) -> set[Table]:
                 SupersetError(
                     error_type=SupersetErrorType.QUERY_SECURITY_ACCESS_ERROR,
                     message=__(
-                        f"You may have an error in your SQL statement. {message}"
-                    ),
```

Comment:
I wonder if @john-bodley knows a good way to lint against this. According to GPT, we can write a custom pylint checker, but it seems like that would be a first for this codebase. Something off the shelf to prevent this would be nice.

Reply:
@rusackas we have custom (WIP) Pylint checkers. I guess one would need to add one to check whether there's an f-string being used within a translation `__(...)`.

Reply:
Oh, could you point me to where those live, @john-bodley, and I'll try to GPT us out of this issue?